pom/firstTask.js

```diff
@@ -0,0 +1,126 @@
+const Page = require('./page');
+
+
+class Condition extends Page {
+    get GoogleSearchInput(){
+        return $('//input[@class = "devsite-search-field devsite-search-query"]')
+    }
+    get GoogleCalculatorPage(){
+        return $('//a[@data-ctorig = "https://cloud.google.com/products/calculator"]')
+    }
+    get GoogleCalculatorPageTitle(){
+        return $('//h2["Google Cloud Pricing Calculator"]');
+    }
+    get firstFrame() {
+        return $('#cloud-site > devsite-iframe > iframe');
+      }
+    
+    get secondFrame() {
+        return $('#myFrame');
+      }
+    async goToCalculatorFrame() {
+        await this.firstFrame.waitForDisplayed();
+        await browser.switchToFrame(await this.firstFrame);
+        await this.secondFrame.waitForDisplayed();
+        await browser.switchToFrame(await this.secondFrame);
+      }
+    get FirstNumberInput(){
+      return $('#input_92')
+    }
+    get SeriesDropDownMenuLabel(){
+      return $('#select_value_label_87')
+    }
+    get SeriesDropDownMenuOption(){
+      return $('//div[contains (text(), "N1")]');
+    }
+    get ComputerOptionDropDownMenu(){
+      return $('#select_value_label_88');
+    }
+    get ComputerOptionDropDownMenuOption(){
+      return $('//div[contains (text(), "n1-standard-8 (vCPUs: 8, RAM: 30GB")]')
+    }
+    get CheckBoxGpu(){
+      return $('//div[contains (text(), "Add GPUs.")][1]')
```

Comment:
Poor locator. Can be used for instance: `md-checkbox[ng-model="listingCtrl.computeServer.addGPUs"]`.